webapp/src/components/history.tsx

```diff
@@ -41,6 +42,10 @@ export function History() {
     list(1);
   }, [form]);
 
+  useEffect(() => {
+    sessionStorage.setItem('historyForm', JSON.stringify(form));
```

Comment:
why not just save the username? does the form save all the history data too!

Reply:
No, its just saving the search fields `RoutedTo`, `User` and `QueryId`. I thught what if in the future we need to save the other fields as well just like username when someone goes away from the page and come back.

Currently other fields are not using the `initValue` attribute so we are not setting any default values, which makes sense for `RoutedTo` and `QueryId`.

I think for now like you suggested we can just save the username.